components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -63,7 +64,7 @@
   const defaultValueMap: Partial<Manifest> = {
     allow_booking: false,
     attendees_to_show: 5,
-    capacity: 1,
+    capacity: null,
```

Comment:
Defaulting to `null` instead of `1` here -- capacity having a numeric value throws an error upon entering `email_ids`, as those properties are mutually exclusive for now.